.github/actions/vtk-install-dep/action.yml

```diff
@@ -107,7 +111,7 @@ runs:
         -DVTK_MODULE_ENABLE_VTK_FiltersGeneral=YES
         -DVTK_MODULE_ENABLE_VTK_FiltersGeometry=YES
         -DVTK_MODULE_ENABLE_VTK_IOCityGML=YES
-        -DVTK_MODULE_ENABLE_VTK_IOExodus=YES
+        -DVTK_MODULE_ENABLE_VTK_IOExodus=${{ inputs.cpu == 'x86_64' && 'YES' || 'NO' }}
```

Comment:
ideally open an issue about exodus not being support on arm

Reply:
and link it in this file

Reply:
That's not new though. HDF5 fails to build on Android (ARM) and wasm too.  
I think the reason is HDF5 generate an executable and execute it during the configuration, but execution fails because we are cross-compiling.

Reply:
yes but we are not shipping anything android or wasm, but we will ship arm, so user we want to know that exodus will not work in that case.

Reply:
ok